Jenkinsfile-Agent-Test-2

```diff
@@ -87,6 +87,12 @@ pipeline {
                             }
                         }
 
+                        stage('spring-webflux 2.x (18)') {
```

Comment:
@zhaoyuguang You forgot the total case number

Reply:
Thanks